tools/cask/main.go

```diff
@@ -378,8 +378,8 @@ func handleSignals(ctx context.Context, id string, dockerClient *client.Client)
 	go func() {
 		sig := <-gracefulExit
 		fmt.Printf("Received %v, stopping container\n", sig)
-		timeout := 0 * time.Second
-		err := dockerClient.ContainerStop(ctx, id, &timeout)
+		timeout := 0
+		err := dockerClient.ContainerStop(ctx, id, container.StopOptions{Timeout: &timeout})
```

Comment:
Cool, I see this came in as we hit docker v22 ([ref](https://github.com/moby/moby/pull/43206/commits/90de570cfa5f6de0b02bdf2e794627fbf5b0dfc8)) :+1: